### PR TITLE
ISPN 11 Query support

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

File: extensions/query/src/test/java/org/radargun/util/QueryTraitRepository.java [hunk 1]
```diff
@@ -61,93 +61,93 @@ public Builder(ConcurrentHashMap cache) {
          }
 
          @Override
-         public Query.Builder subquery() {
+         public org.radargun.traits.Query.Builder subquery() {
```
Comment (alanfx) [inline extensions/query/src/test/java/org/radargun/util/QueryTraitRepository.java:64]: Why is this needed? Can't you import the class with the name `Query` to avoid this on line 3 above?

Comment (diegolovison) [reply extensions/query/src/test/java/org/radargun/util/QueryTraitRepository.java]: It should be good now

**Comment by alanfx** [comment]
Looks good, @diegolovison! Merging
